pkg/ibmvpc/vpc.go

```diff
@@ -57,7 +57,11 @@ func (r *ReservedIP) Name() string {
 	return getNodeName(r.vpe, r.Address())
 }
 
-// ReservedIP implements vpcmodel.Node interface
+func (r *ReservedIP) ExtendedName(c *vpcmodel.VPCConfig) string {
+	return r.ExtendedPrefix(c) + getNodeName(r.vpe, r.Address())
```

Comment:
```suggestion
	return r.ExtendedPrefix(c) + r.Name()
```
avoid duplicating the code implemented within `Name()` method